enterprise/views.py

```diff
@@ -477,6 +482,10 @@ class CourseEnrollmentView(View):
             'Not eligible for a certificate; does not count toward a MicroMasters'
         ),
         'continue_link_text': _('Continue'),
+        'level_text': _('Level'),
+        'effort_text': _('Effort'),
+        'effort': _('{hours} hours per week, per course'),
```

Comment:
@replaceafill IIUC, `effort` is used to generate `course_effort` below. You could rename it to `course_effort_text` to be more precise, and to stick with the `*_text` naming scheme that other context variables are using.